subprojects/packagefiles/arrow-cpp/meson.build

```diff
@@ -0,0 +1,100 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The wrap's patch zip will automatically include a `LICENSE.build` file declaring an MIT license unless the `arrow-cpp` subdirectory has a `LICENSE.build` file of its own.  One should be added, with appropriate license text.

Reply:
The new `LICENSE.build` includes licenses for a lot of files that aren't in WrapDB.  The file is only meant to cover the `arrow-cpp` WrapDB subdirectory, not the entire upstream project.

Reply:
Ah OK - I just copied it from the Arrow codebase. I'll strip it down to just the core Apache license then